FirebaseGoogleAuthUI/Sources/Public/FirebaseGoogleAuthUI/FUIGoogleAuth.h

```diff
@@ -85,6 +89,12 @@ __attribute__((deprecated("Instead use initWithAuthUI:")));
 - (instancetype)initWithScopes:(NSArray <NSString *> *)scopes
 __attribute__((deprecated("Instead use initWithAuthUI:permissions:"))) NS_DESIGNATED_INITIALIZER;
 
+/** @fn initWithAuthUI:scopes:
+    @brief Requests the scopes in the `scopes` array.
+ */
+- (void)requestScopesWithPresentingViewController:(UIViewController *)presentingViewController
+                                       completion:(FUIAuthProviderSignInCompletionBlock)completion;
```

Comment:
@rosalyntan does this approach make sense?

Reply:
I think so -- so will the developer flow for this be to call `signInWithDefaultValue` first to authenticate, and then `requestScopesWithPresentingViewController` to request the appropriate scopes?

Reply:
Yes.